### PR TITLE
Add proper pagination handling for all the things

### PR DESCRIPTION
**Pull request by odinuge**

### CODE REVIEW

**Comment by odinuge** [comment]
This now adds nice pagination to emailLists, groupMembers, emailUsers and articles. The tables in the first three of them also supports real filtering using the backend for querying. There might be some rough edges, but feel free to test. The reducer code might benefit from some tests, but that is not the first prio right now... :sweat_smile:

**Comment by odinuge** [comment]
Screenshot from 2020-10-19 17-15-01
Screenshot from 2020-10-19 17-15-28
Screenshot from 2020-10-19 17-15-10

File: app/utils/createEntityReducer.js [hunk 1]
```diff
@@ -133,8 +148,24 @@ export function deleteEntities(deleteTypes: ?EntityReducerTypes) {
 
     if (!resultId) return state;
 
+    const paginationNext = Object.keys(state.paginationNext).reduce(
```
Comment (odinuge) [inline app/utils/createEntityReducer.js:151]: Best code ever :tm:

**Review by liseph — APPROVED** [review]
lgtm :rip:

File: app/routes/admin/email/components/EmailRoute.js [hunk 2]
```diff
@@ -13,7 +13,11 @@ const EmailPage = ({ children }: Props) => (
   <Content>
     <NavigationTab title="Epost">
       <NavigationLink to={'/admin/email'}>Lister</NavigationLink>
-      <NavigationLink to={'/admin/email/users'}>Brukere</NavigationLink>
+      <NavigationLink
+        to={'/admin/email/users?filters={"internalEmailEnabled"%3A"true"}'}
```
Comment (SmithPeder) [inline app/routes/admin/email/components/EmailRoute.js:17]: ⭕

**Review by SmithPeder — APPROVED** [review]

**Pull request merged**